sdk/eventhubs/test-resources.json

```diff
@@ -0,0 +1,430 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "String"
+        },
+        "storageEndpointSuffix": {
+            "type": "string",
+            "defaultValue": "core.windows.net"
+        },
+        "eventHubName": {
+            "type": "string",
+            "defaultValue": "javaeventhub"
+        },
+        "eventHubNamespaceHostNameSuffix": {
+            "type": "string",
+            "defaultValue": "servicebus.windows.net"
+        },
+        "testApplicationOid": {
+            "type": "string",
+            "metadata": {
+                "description": "The principal to assign the role to. This is application object id."
+            }
+        },
+        "tenantId": {
+            "type": "string",
+            "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+            "metadata": {
+                "description": "The tenant ID to which the application and resources belong."
+            }
+        },
+        "testApplicationId": {
+            "type": "string",
+            "metadata": {
+                "description": "The application client ID used to run tests."
+            }
+        },
+        "testApplicationSecret": {
+            "type": "string",
+            "metadata": {
+                "description": "The application client secret used to run tests."
+            }
+        },
```

Comment:
This closes the `testApplicationSecret` object, but not the `parameters` object, so the variables section looks like another parameter to the script. 

This line should be `}},` to close the `testApplicationSecret` object and close the `parameters` section.